payload/src/main.rs

```diff
@@ -28,4 +28,7 @@ fn main() {
     for aux in &mut reader {
         println!("aux: {:?}", aux);
     }
+
+    println!();
+    println!("that's a nice TNETENNBA!");
```

Comment:
```suggestion
    println!("Good morning, that's a nice tnetennba!");
```

Reply:
The capitalisation is disputed: I'll amend slightly.  The salutation should, indeed, be included.